packages/kotti-ui/source/kotti-comment/KtComment.vue

```diff
@@ -95,75 +104,74 @@ export default {
 		KtCommentInput,
 	},
 	props: makeProps(KottiComment.propsSchema),
-	data() {
-		return {
-			showInlineReply: false,
-			replyToText: null,
-			replyToUserId: Number,
```

Comment:
one day I'm going to stroke while reading one of those gems

Reply:
:pepe-hug: